### PR TITLE
Update Set-CsCallQueue.md

### PR DESCRIPTION
**Pull request by CLYVR**
Removing restriction note on -OboResourceAccountIds as this feature now works all call queues
@get-itips @dariomws @tseward @tiburd  Please and thanks!

### CODE REVIEW

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/Set-CsCallQueue.md [hunk 1]
```diff
@@ -146,7 +146,6 @@ Accept wildcard characters: False
 ### -OboResourceAccountIds
 The OboResourceAccountIds parameter lets you add resource account with phone number to the Call Queue. The agents in the Call Queue will be able to make outbound calls using the phone number on the resource accounts.  This is a list of resource account GUIDs.
 
-Only Call Queue managed by a Teams Channel will be able to use this feature. For more information, please refer to [Manage your support Call Queue in Teams](https://support.microsoft.com/office/manage-your-support-call-queue-in-teams-9f07dabe-91c6-4a9b-a545-8ffdddd2504e).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsCallQueue.md:149]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@CLYVR, please also remove the extra blank line. Many thanks.

**Comment by tiburd** [comment]
@CLYVR 
Hi, Colin,
Please see Johan's comment above, and then @ mention me again when this PR is ready to merge (or merge it, if you have perms to do so).
Thank you!

**Comment by CLYVR** [comment]
@tiburd Blank line removed.  I don't have the perms to merge myself.  Thank you!

**Pull request merged**